### DISCUSSION TITLE
[Bug Report][3.4.0] icons became weird when open previous page from url

### DISCUSSION THREAD

**Discussion by Mukhammad-Umar** [vuetifyjs/vuetify#18670] [Q&A]
### Environment
**Vuetify Version:** 3.4.0
**Vue Version:** 3.2.0
**Browsers:** Chrome 119.0.0.0
**OS:** Windows 10

### Steps to reproduce
It happens sometimes when I go next page and then come back previous page through url, icons become weird like O symbols and it happen with selects, buttons and components which have icons. I changed version of vuetify to 3.4.0 and mdi to 7.1.96 but nothing changed.

### Expected Behavior
show expected icons instead of O ones

### Actual Behavior
showing O icons and after refresh shows usual ones

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqNUk1PAjEQ/StjL6uJCwocDFlI/AMevAKHsp2Vxm5b2+4GA/x3p7vgFqOJp8681/no61sdmHfl+NnaUdsgm7MiYG0VD7hca4Cizbm1XdglpdGBS43uDPXgDsv3rdlDm9dGoFqsGX9ziGLNoDTKOAKkrgylOykwF0g9lF8W46E27edRYRkuACRtJS23ZgNTKu494XXIJymejhlQbXLBA88D7gMVvRiI6VW/q20HfB4Hx0HdmTICtZfhk6jS1JbT2gnZcie5jrNMExTJRookTV2jMDZdtbBYws1NC8cjZK/40UiHAiqJSmSboaQTrBcnlWsb9K+KzP7zMkk/Gq8LmVvV/P20yrjAtwp/bEOzL+aIn3nljoj05inGiako9aWTNoDH0PTekrWl/nAAhxWcoHKmhoz8mMXrcXntA0TtYRGv3GZd/JjdDWzvuDMfXIMJ1/3amVpdau+hjybf0TTbUFEx7tejxdjpnk1H09ETi+ds9MA2X8QY+To=)

### Other comments
I published in stackoverflow with images of icons  https://stackoverflow.com/questions/77472515/vuetify-shows-weird-icons

**Comment by KaelWD** [comment]
https://github.com/Templarian/MaterialDesign/issues
